### PR TITLE
feat (ui): workflow join

### PR DESCRIPTION
**Pull request by sguiheux**

### CODE REVIEW

File: ui/src/assets/i18n/fr.json [hunk 1]
```diff
@@ -390,13 +390,20 @@
   "workflow_added" : "Workflow créé",
   "workflow_create" : "Création d'un workflow",
   "workflow_description" : "Description du workflow",
+  "workflow_join_delete" : "Suppression de la jointure",
+  "workflow_join_delete_alert" : "ATTENTION, cela va supprimer la jointure et tout ce qu'elle déclenche",
+  "workflow_join_src_delete_alert" : "Voulez-vous vraiment supprimer ce lien pipeline-join ?",
```
Comment (fsamin) [inline ui/src/assets/i18n/fr.json]: pipeline-join ?

**Comment by fsamin** [comment]
build is failed

File: ui/src/assets/i18n/en.json [hunk 2]
```diff
@@ -390,13 +390,20 @@
   "workflow_added" : "Workflow added",
   "workflow_create" : "Create a workflow",
   "workflow_description" : "Workflow description",
+  "workflow_join_delete" : "Delete join",
+  "workflow_join_delete_alert" : "BE CAREFUL, this will remove the join and all his children",
+  "workflow_join_src_delete_alert" : "Do you really want to remove join source pipeline ?",
```
Comment (yesnault) [inline ui/src/assets/i18n/en.json]: space

File: ui/src/assets/i18n/en.json [hunk 3]
```diff
@@ -390,13 +390,20 @@
   "workflow_added" : "Workflow added",
   "workflow_create" : "Create a workflow",
   "workflow_description" : "Workflow description",
+  "workflow_join_delete" : "Delete join",
```
Comment (yesnault) [inline ui/src/assets/i18n/en.json]: Delete the join? (or remove)

File: ui/src/assets/i18n/en.json [hunk 4]
```diff
@@ -390,13 +390,20 @@
...
+  "workflow_join_delete_alert" : "BE CAREFUL, this will remove the join and all his children",
+  "workflow_join_src_delete_alert" : "Do you really want to remove join source pipeline ?",
+  "workflow_join_trigger_title" : "Trigger",
+  "workflow_join_trigger_src_delete" : "Delete source link",
```
Comment (yesnault) [inline ui/src/assets/i18n/en.json]: the

**Pull request merged**